autopush/metrics.py

```diff
@@ -34,6 +31,9 @@ def __init__(self, *args, **kwargs):
     def start(self):
         """Start any connection needed for metric transmission"""
 
+    def make_tags(self, base=None, **kwargs):
```

Comment:
I don't see any real implementations of this method, but it's used a lot. Maybe it should have a default implementation which just passes on the data as-is?

Reply:
yeah, it's a bit of legacy. Originally, we had to convert the tags from a dict to a list. Thanks for spotting, because it reminded me that I needed to make sure that I did the opposite, and reverted things back to a dictionary. If we ever need to use DatadogMetrics again, we can use this to bottleneck metrics back into lists.